run-in-php-docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+args="$@"
+docker exec -ti phphsite_phph-php-fpm_1 bash -c "cd /app; $args"
```

Comment:
This doesn't work unless your user is in the docker group. Might be better just to use `docker-compose exec phph-php-fpm <command>`?